assets/xml/objects/object_harfgibud.xml

```diff
@@ -1,10 +1,13 @@
 ﻿<Root>
+    <!-- Object for the Pamela's father, both human and gibdo form-->
     <File Name="object_harfgibud" Segment="6">
-        <Animation Name="object_harfgibud_Anim_000370" Offset="0x370" />
-        <Animation Name="object_harfgibud_Anim_001138" Offset="0x1138" />
-        <Animation Name="object_harfgibud_Anim_0015D4" Offset="0x15D4" />
-        <Animation Name="object_harfgibud_Anim_001960" Offset="0x1960" />
-        <Animation Name="object_harfgibud_Anim_00260C" Offset="0x260C" />
+
+        <!-- Gibdo Form -->
+        <Animation Name="gHarfgibudLeanForwardAnim" Offset="0x370" />
```

Comment:
Instead of `gHarfgibud` as a prefix, how about `gPamelasFather` or something? We don't need to include the object's name in an object symbol, so I think being clearer is better.